File: SCC0600/exercíciosPy/ex5.py
# ENTRADA:
ano = int(input())

# PROCESSAMENTO:
if ano <= 0:
  res = -1
else:
  # ano > 0
  if ano % 400 == 0 or (ano % 4 == 0 and ano % 100 != 0):
    res = 1
  else:
    res = 0

# SAIDA:
print(res)

File: SCC0600/exerciciosC/ex5.c
#include <stdio.h>

int main (void) {
    int ano; int res = 0;
    scanf("%d", &ano);

    if (ano > 0) {
        if (ano % 400 == 0 || (ano % 4 == 0 && ano % 100 != 0)) {
            res = 1;
        }
    } else {
        res = -1;
    }

    printf("%d\n", res);
}